OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -634,27 +755,26 @@ static bool Kick(S server, Connection conn, Session.Client client, string s)
 				var kickConn = server.Conns.SingleOrDefault(c => server.GetClient(c)?.Index == kickClientID);
 				if (kickConn == null)
 				{
-					server.SendOrderTo(conn, "Message", "No-one in that slot.");
+					server.SendLocalizedMessageTo(conn, KickNone);
 					return true;
 				}
 
 				var kickClient = server.GetClient(kickConn);
 				if (server.State == ServerState.GameStarted && !kickClient.IsObserver)
 				{
-					server.SendOrderTo(conn, "Message", "Only spectators can be kicked after the game has started.");
+					server.SendLocalizedMessageTo(conn, NoKickGameStarted);
 					return true;
 				}
 
-				Log.Write("server", "Kicking client {0}.", kickClientID);
-				server.SendMessage($"{client.Name} kicked {kickClient.Name} from the server.");
-				server.SendOrderTo(kickConn, "ServerError", "You have been kicked from the server.");
+				Log.Write("server", $"Kicking client {kickClientID}.");
+				server.SendLocalizedMessage(ClientKicked, Translation.Arguments("admin", client.Name, "client", kickClient.Name));
+				server.SendOrderTo(kickConn, "ServerError", "you-were-kicked");
```

Comment:
Maybe we should expose the property on the `Server` class so we can reference it directly rather than using `"you-were-kicked"` here?